assets/aws/cmd/update-ami-id/main.go

```diff
@@ -120,18 +122,20 @@ func main() {
 	}
 
 	for _, region := range strings.Split(*regions, ",") {
-		newAMI := imageIDs[region]
+		for _, arch := range []string{"x86_64", "arm64"} {
+			newAMI := imageIDs[region]
 
-		ts := AMIType(*amiType)
-		re, err := regexp.Compile(fmt.Sprintf(`(?m)^# %s v(.*) %s: (ami-.*)$`, region, ts.FriendlyType()))
-		if err != nil {
-			log.Fatalf("invalid regexp for region %q type %q: %v", region, *amiType, err)
-		}
+			ts := AMIType(*amiType)
+			re, err := regexp.Compile(fmt.Sprintf(`(?m)^# %s v(.*) %s %s: (ami-.*)$`, region, arch, ts.FriendlyType()))
+			if err != nil {
+				log.Fatalf("invalid regexp for region %q type %q arch %q: %v", region, *amiType, arch, err)
+			}
 
-		repl := fmt.Sprintf(`# %s v%s %s: %s`, region, *version, ts.FriendlyType(), newAMI)
-		md = re.ReplaceAll(md, []byte(repl))
+			repl := fmt.Sprintf(`# %s v%s %s %s: %s`, region, *version, arch, ts.FriendlyType(), newAMI)
+			md = re.ReplaceAll(md, []byte(repl))
```

Comment:
IIRC this is only going to replace existing entries in AMIS.md. Who/what is going to add the initial set of arm64 AMIs?

Reply:
Yeah, I was waiting for #36112 to land first, but I guess I can just add placeholders that will be updated in the meantime.

Reply:
Placeholders added.